bread/layout/base.py

```diff
@@ -137,6 +138,8 @@ def resolve(self, context):
             value = self.formatter(value)
         if isinstance(value, models.Manager):
             value = ", ".join([str(x) for x in value.all()])
+        if isinstance(value, str):
+            value = linebreaksbr(value, autoescape=False)
```

Comment:
Actually `autoescape=True` seems to lead to exactly the same result... Not sure which one makes more sense.

Reply:
Okay, then we can leave it with the default I guess.